Screenplay.md

```diff
@@ -0,0 +1,15 @@
+Screenplay
+==========
+General Intro
+
+## Explaining the code base
+
+## Explaining the specs written by the BA
+
+## Explain how the specs are written. The various patterns used for writing the spec.
```

Comment:
Is this how the specs are "instrumented" to turn the examples into tests? 

ie. maybe we talk of the 3 stages:

1. authoring the spec
2. instrumenting the spec
3. writing the test code